API/Controllers/ProjectController.cs

```diff
@@ -56,23 +57,56 @@ public ProjectController(IProjectService projectService, IUserService userServic
         /// <summary>
         ///     Get all projects.
         /// </summary>
+        /// <param name="parameters">The parameters to narrow down the projects</param>
         /// <returns></returns>
         [HttpGet]
-        public async Task<IActionResult> GetAllProjects()
+        public async Task<IActionResult> GetAllProjects([FromQuery] RequestParamsResource parameters)
```

Comment:
parameter name is not clear

Reply:
Is the description in the documentation above clear? If it is, this should be fine. If it is not, I will change it.